### ISSUE TITLE
json comparison on --- response_body 

### ISSUE THREAD

**Issue by skelzer** [openresty/test-nginx#159]
Hello!,

First of all, sorry if this is a dumb question, but I have never used nginx or Perl before, so I'm a bit at a loss.

I am trying to validate an endpoint that should serve a JSON as a response.  Since JSONs are by definition unsorted, how should I go about validating this response? Is this even possible in this framework?

Thanks :)

**Comment by frankgreco** [comment]
This was a life saver. Could we add this to this guide: https://openresty.gitbooks.io/programming-openresty/content/testing/test-file-layout.html

**Comment by zhuizhuhaomeng** [comment]
@frankgreco would you please create a PR for your proposal?

https://github.com/openresty/programming-openresty/

**Comment by frankgreco** [comment]
@zhuizhuhaomeng yes but first this actually isn't working for me. `cpan install Test::Nginx` installs `0.30` which I'm guessing doesn't contain the ability to do .
```
--- response_body_json_sort
--- response_body
```
